src/routes/(app)/members/[studentId]/edit-bio/+page.server.ts

```diff
@@ -1,13 +1,12 @@
-import apiNames from "$lib/utils/apiNames";
 import { memberSchema } from "$lib/zod/schemas";
 import { error, fail } from "@sveltejs/kit";
 import { message, superValidate } from "sveltekit-superforms/server";
 import type { Actions, PageServerLoad } from "./$types";
-import { authorize } from "$lib/utils/authorization";
+import { redirect } from "$lib/utils/redirect";
 
 export const load: PageServerLoad = async ({ locals, params }) => {
   const { prisma, user } = locals;
-  authorize(apiNames.MEMBER.UPDATE, user);
+  if (!user || !user.memberId) throw redirect(302, "/home");
```

Comment:
Why? I think it's weird to redirect someone to home screen instead of just showing error. It can make the page seem confusing

Reply:
You've also removed the verification that a user can actually edit this person's bio.

Reply:
But I see, the old system seems to not check if params.studentId === user.studentId, in which case you should be able to edit. Otherwise you cannot edit your own bio.